charts/common/test-values.yaml

```diff
@@ -202,3 +202,7 @@ env:
     type: parameterStore
     name: example-db
     parameter_name: DB_PASSWORD
+
+httpBasicAuth:
+  enabled: true
+  usersList: "user1:$apr1$VE.bbkmA$JXZDn5ch.5YY2WmsGFUOt1" # user1:password
```

Comment:
This is just an example that works. `hashed_pasword1` doesn't work as an actual hashed password, that's why I'm using one of the possible hashes for the word `password`, so that it can actually be tested.

Since this is just a Helm chart, a template, there's no "real" value associated with it.